client/my-sites/email/email-management/gsuite-users-card/style.scss

```diff
@@ -25,6 +25,10 @@
 		&:last-of-type {
 			border-bottom: 1px solid var( --color-neutral-0 );
 		}
+
+		& .gsuite-user-item__email {
+			font-family: $serif;
+		}
```

Comment:
Why did you add this rule here instead of [here](https://github.com/Automattic/wp-calypso/blob/bee6b9c3a923fc54f45bed959c25c1f896c8fc42/client/my-sites/email/email-management/gsuite-user-item/style.scss#L10)?

Reply:
I just realized now that there is an inner unit for the `GSuiteUserItem` component. Fixing ...